### PR TITLE
removed Jupyter link from Community Data and Published

### PR DESCRIPTION
**Pull request by van-go**
## Overview: ##
Removed Jupyter link from "Published" and "Community Data" 

## PR Status: ##

* [X] Ready.
* [ ] Work in Progress.
* [ ] Hold.

## Related Jira tickets: ##

* [DES-2281](https://jira.tacc.utexas.edu/browse/DES-2281)

## Summary of Changes: ##
In the file-listing template, I commented out the section that put the Jupyter link in the table.

## Testing Steps: ##
1. Community Data --> Jupyter Notebooks. Notice no "Jupyter" links in the "Open with" column for files that end with pynb.
2. Published --> keyword search: Jupyter. Notice no "Jupyter" links in the "Open with" column for files that end with pynb.

## UI Photos:
Screen Shot 2022-07-19 at 10 52 03 AM

### CODE REVIEW

File: designsafe/static/scripts/data-depot/components/data-depot-listing/files-listing.template.html [hunk 1]
```diff
@@ -80,10 +80,11 @@
                     <td data-ng-if="!item.lastModified"class="test-date-missing">--</td>
                     <td data-ng-if="$ctrl.operationLabel" class="test-op-label">
                         <button class="btn btn-primary test-op-btn" ng-if="item.type==='dir'" ng-click="$ctrl.operation({file: item})">{{$ctrl.operationLabel}}
-
-                        </button></td>
+                        </button>
+                    </td>
                     <td data-ng-if="!$ctrl.FileListingService.listings.main.loading && $ctrl.hasNotebooks()">
```
Comment (Strmiska) [inline designsafe/static/scripts/data-depot/components/data-depot-listing/files-listing.template.html:85]: The ticket for this issue says the link is broken, are we removing the link entirely or are we supposed to restore the functionality? If we're removing this, we may want to remove the table column as well.

Comment (van-go) [reply designsafe/static/scripts/data-depot/components/data-depot-listing/files-listing.template.html:85]: Good point. I'm not sure if that column is used for anything else. I'll poke around and check. If it's not used, I'll remove it.

File: designsafe/static/scripts/data-depot/components/data-depot-listing/files-listing.template.html [hunk 2]
```diff
@@ -22,7 +22,7 @@
                     <th>Size</th>
                     <th>Last modified</th>
                     <th ng-if="$ctrl.operationLabel">Operation</th>
-                    <th ng-if="!$ctrl.FileListingService.listings.main.loading && $ctrl.hasNotebooks()">Open with</th>
+                    <!-- <th ng-if="!$ctrl.FileListingService.listings.main.loading && $ctrl.hasNotebooks()">Open with</th> -->
```
Comment (Strmiska) [inline designsafe/static/scripts/data-depot/components/data-depot-listing/files-listing.template.html:25]: Let's just cut these out. We can always check the repo history if we need to add them back for any reason.
```suggestion
```

File: designsafe/static/scripts/data-depot/components/data-depot-listing/files-listing.template.html [hunk 3]
```diff
@@ -80,11 +80,12 @@
...
-
-                        </button></td>
-                    <td data-ng-if="!$ctrl.FileListingService.listings.main.loading && $ctrl.hasNotebooks()">
-                        <a href="{{$ctrl.openInApp(item)}}" target="_blank" rel="noopener" data-ng-if="$ctrl.isNotebook(item)">Jupyter</span>
+                        </button>
                     </td>
+                    <!-- <td data-ng-if="!$ctrl.FileListingService.listings.main.loading && $ctrl.hasNotebooks()"> -->
+                        <!-- <a href="{{$ctrl.openInApp(item)}}" target="_blank" rel="noopener" 
+                        data-ng-if="$ctrl.isNotebook(item)">Jupyter</span> -->
+                    <!-- </td> -->
```
Comment (Strmiska) [inline designsafe/static/scripts/data-depot/components/data-depot-listing/files-listing.template.html:88]: ✂️ 
```suggestion
```

**Review by Strmiska — APPROVED** [review]
LGTM!

**Pull request merged**